iOS/Flows/Home/_New/new_ConversationViewController.swift

```diff
@@ -61,10 +63,25 @@ class new_ConversationViewController: FullScreenViewController, CollectionViewIn
 
         self.view.addSubview(self.blurView)
         self.view.addSubview(self.collectionView)
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+
+        self.blurView.expandToSuperviewSize()
+        self.collectionView.expandToSuperviewSize()
+    }
 
-        self.subscribeToUpdates()
-        self.setupInputHandlers()
-        self.loadMessages()
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        once(caller: self, token: "intializeCollectionView") {
+            Task {
+                self.setupInputHandlers()
+                await self.loadInitialMessages()
+                self.subscribeToConversationUpdates()
+            }
```

Comment:
Only calling the initializer functions once.

Reply:
why not put this in viewdidload?